kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/properties-template.properties

```diff
@@ -0,0 +1,38 @@
+DOMAIN_NAME=%DOMAIN_NAME%
+ADMIN_PORT=%ADMIN_PORT%
+ADMIN_NAME=%ADMIN_SERVER_NAME%
+ADMIN_HOST=wlsadmin
+MANAGED_SERVER_PORT=%MANAGED_SERVER_PORT%
+MANAGED_SERVER_NAME_BASE=%MANAGED_SERVER_NAME_BASE%
+CONFIGURED_MANAGED_SERVER_COUNT=%CONFIGURED_MANAGED_SERVER_COUNT%
+CLUSTER_NAME=%CLUSTER_NAME%
+DEBUG_PORT=8453
+DB_PORT=1527
+DEBUG_FLAG=true
+DERBY_FLAG=true
+PRODUCTION_MODE_ENABLED=%PRODUCTION_MODE_ENABLED%
+CLUSTER_TYPE=%CLUSTER_TYPE%
+JAVA_OPTIONS=%JAVA_OPTIONS%
+T3_CHANNEL_PORT=%T3_CHANNEL_PORT%
+T3_PUBLIC_ADDRESS=%T3_PUBLIC_ADDRESS%
+#Derby Data Source parameters
+dsname=DockerDS
+dsdbname=DerbyDB;create=true
+dsjndiname=/DockerDS
+dsdriver=org.apache.derby.jdbc.ClientXADataSource
+dsurl=jdbc:derby://localhost:1527/DerbyDB;ServerName=localhost;databaseName=DerbyDB;create=true
+dsusername=dbuser
+dspassword=dbpassword
+dstestquery=SQL SELECT 1 FROM SYS.SYSTABLES
+dsinitalcapacity=0
+dsmaxcapacity=15
+#Oracle database settings
```

Comment:
If these commented out parameters are not going to be used, shall we remove them?